lms/djangoapps/verify_student/models.py

```diff
@@ -1186,3 +1186,64 @@ def check_user_skipped_reverification_exists(cls, user, course_id):
             Boolean
         """
         return cls.objects.filter(user=user, course_id=course_id).exists()
+
+
+class CreditCourse(models.Model):
+    """Model for tracking the credit course."""
+
+    course_key = CourseKeyField(max_length=255, db_index=True, unique=True)
+
+
+class CreditProvider(models.Model):
+    """This model represents an institution that can grant credit for a course.
+    each provider is identified by unique id e.g ASU
+    """
+    provide_id = models.CharField(max_length=255, db_index=True, unique=True)
```

Comment:
it doesn't seem like this field is needed at all, according to the spec document, which only wants a primary key `id` (which django will create automatically).  the one thing i found confusing about the spec is that it specifies for `id` to be a char - @wedaly could you clarify?